explorer/explorerroutes.go

```diff
@@ -540,7 +550,7 @@ func (exp *explorerUI) Blocks(w http.ResponseWriter, r *http.Request) {
 	}
 
 	rows, err := strconv.Atoi(r.URL.Query().Get("rows"))
-	if err != nil || (rows < minExplorerRows && rows == 0) {
```

Comment:
I couldn't figure out a reason why this was here. Seems that it should be one or the other, but not both.

Reply:
There's not much value in enforcing at least `minExplorerRows` either.  How about:

```go
if err != nil || rows < 1 {`
```